debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/AbstractReader.java

```diff
@@ -48,7 +48,7 @@
     private final Metronome metronome;
     private final AtomicReference<Runnable> uponCompletion = new AtomicReference<>();
     private final Duration pollInterval;
-
+    private boolean stopping;
```

Comment:
There already is `state()` and `State.STOPPING` in particular. This should be consolidated instead of adding this field here.